### ISSUE TITLE
How to extract the text content of web page.

### ISSUE THREAD

**Issue by kunal0810** [microsoft/playwright#2713]
i have used the function page.content(); but it returns the raw html .
is there any function which return only the text content display on the web page.

**Comment by arjun27** [comment]
Would this one of these work for you? These methods fetch inner text/text content from a selector, and I'm passing `body` to get the text inside the `<body>` tag

```js
await page.innerText('body');

await page.textContent('body');
```

**Comment by kunal0810** [comment]
Hi Arjun , thanks for responding.
i have tried both above method you suggest but getting an error "page.innerText is not a function"  same error for function textcontent too.

**Comment by arjun27** [comment]
what version of Playwright are you using? These APIs are available since 1.1

**Comment by kunal0810** [comment]
Thanks Arjun, yep i was using some old version . i have updated that. it resolved my problem.
